docs/streams/developer-guide/config-streams.html

```diff
@@ -92,7 +93,6 @@
               <li><a class="reference internal" href="#task-assignor-class" id="id39">task.assignor.class</a></li>
               <li><a class="reference internal" href="#topology-optimization" id="id31">topology.optimization</a></li>
               <li><a class="reference internal" href="#windowed-inner-class-serde" id="id38">windowed.inner.class.serde</a></li>
-              <li><a class="reference internal" href="#log-summary-interval-ms" id="id40">log.summary.interval.ms</a></li>
```

Comment:
This is a new config and it should be added at the right place. (The original PR was just a little "sloppy"...)